transforms/csdl/preprocess_csdl_test_output.xml

```diff
@@ -217,6 +217,19 @@
             </Record>
           </Annotation>
         </NavigationProperty>
+        <NavigationProperty Name="synchronization" Type="graph.synchronization" ContainsTarget="true" />
+      </EntityType>
+      <EntityType Name="synchronization" BaseType="graph.entity">
+        <Property Name="secrets" Type="Collection(graph.synchronizationSecretKeyStringValuePair)" />
+        <NavigationProperty Name="jobs" Type="Collection(graph.synchronizationJob)" ContainsTarget="true" />
+      </EntityType>
+      <EntityType Name="synchronizationJob" BaseType="graph.entity">
```

Comment:
Shouldn't the output have the annotation present here?

Reply:
I've made the correction